iCubGenova09/calibrators/left_arm-calib.xml

```diff
@@ -16,23 +16,25 @@
 <param name="calibration2">	       0           0           0          0        16384      0       0        0       0       9102    9102    9102    9102    9102    9102    10000    </param>
 <param name="calibration3">           0           0           0          0           0       0       0       0       0      -1        1       -1      1      -1       1       1      </param>
 <param name="calibration4">           0           0           0          0           0       0       0       1948    325     250     478     243     483     255     480     725     </param>
 <param name="calibration5">           0           0           0          0           0       0       0       2073    2730    9       99       15      8       30      45      125     </param>
-<param name="calibrationZero">        0           0           0          0           0       0       0       0       0       0       0       0       0       0       0       0       </param>
-<param name="calibrationDelta">       14          8           0          0           0       0       0       0       0       0       0       0       0       0       0       0       </param>
+<param name="calibrationZero">        36.2       -8.4        -51.3       -5           0       0       0       0       0       0       0       0       0       0       0       0       </param>
+<param name="calibrationDelta">       0           0           0          0           0       0       0       0       0       0       0       0       0       0       0       0       </param>
                                                                                                                              
-<param name="startupPosition">        -35        30           0         45           0       0       0       25      45      0       0       0       0       0       0       0       </param>
+<param name="startupPosition">        34        50            0        10           -75       0       0       25      45      0       0       0       0       0       0       0       </param>
 <param name="startupVelocity">        10.0       10.0         10.0      10.0         30      30      30      60      100     100     100     100     100     100     100     100     </param>
 <param name="startupMaxPwm">          8000       8000         8000      8000         2000    0       0       0       0       0       0       0       0       0       0       0       </param>
 <param name="startupPosThreshold">    2          2            2          2           90      90      90      90      90      90      90      90      90      90      90      90      </param>
 </group>
 
-<param name="CALIB_ORDER"> (1) (0) (2) (3) (4) (5 6) (7) (8 9) (11) (13) (10) (12) (14) (15) </param> <!-- removed joint 14 -->
+<param name="CALIB_ORDER"> (2) (0) (1) (4) (3) (5 6) (8 9) (11) (13) (10) (12) (14) (15) </param> <!-- removed joint 14 -->
```

Comment:
I think that the joint ``(7)`` disappeared

Reply:
yes, sorry.. gonna fix